include/vara/Feature/Feature.h

```diff
@@ -105,14 +108,22 @@ class Feature {
     Relationships.push_back(std::move(Relationship));
   }
 
+  [[nodiscard]] std::optional<Location> getLocation() const {
+    return Loc ? std::optional{Location(Loc->getPath(), Loc->getStart(),
+                                        Loc->getEnd())}
+               : std::nullopt;
```

Comment:
```suggestion
    return Loc;
```
Just return the optional, a copy will be made automatically.